pkgs/development/mobile/androidenv/test-suite.nix

```diff
@@ -1,9 +1,13 @@
-{ stdenv, callPackage }:
+{}:
 let
-  examples-shell = callPackage ./examples/shell.nix {};
-  examples-shell-with-emulator = callPackage ./examples/shell-with-emulator.nix {};
+  pkgs = import ../../../.. {
+    config.allowUnfree = true;
```

Comment:
I don't think this is ok

Reply:
Because of potential licensing problems or is it a programming issue?

You can check in the check list below that it ran the tests in x86_64-linux successfully, so at least it works!

So I guess it's a licensing issue. But what's the resolution? How could we test this stuff?

Reply:
I removed that line, so we have to checkout locally to run the tests, which is not ideal.